packages/diagrams/frontend/sirius-components-diagrams/src/renderer/dropNode/useDropDiagramStyle.tsx

```diff
@@ -22,22 +22,24 @@ export const useDropDiagramStyle = (): useDropDiagramStyleValue => {
   const { droppableOnDiagram } = useContext<DropNodeContextValue>(DropNodeContext);
   const { getNodes } = useStore();
   const theme = useTheme();
+  const [background, setBackground] = useState<string | number>(theme.palette.background.default);
```

Comment:
Why can that be a number?

Reply:
It's the definition of background in React.CSSProperties.
It can be a string or a number